Bogcha.DataAccess/Repositories/ActivityManagementRepositories/IActivityManagementRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Bogcha.Domain.Entities;
+
+namespace Bogcha.DataAccess.Repositories.ActivityManagementRepositories
+{
+    public interface IActivityManagementRepository
+    {
+
+        public ValueTask<bool> CreateAsync(ActivityManagement activityManagement);
+        public ValueTask<bool> updateAsync(int id, ActivityManagement activityManagement);
```

Comment:
```suggestion
        public ValueTask<bool> updateAsync(int id, ActivityManagement activityManagement);
```
update katta harfda bo'lsin

Reply:
o'zgartirish kiritdim